src/main/java/sam/Item.java

```diff
@@ -0,0 +1,65 @@
+package sam;
+/**
+ * Represents an item in a list.
+ */
+public class Item {
+
+    private final String name;
+    private Boolean isDone;
+
+    /**
+     * Constructs a new Item object with the specified name.
+     *
+     * @param newname the name of the item
+     */
+    public Item(String newname) {
+        this.name = newname;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks the item as done.
+     */
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Marks the item as undone.
+     */
+    public void markAsUndone() {
```

Comment:
I didn't like the name of this method since it is too similar to "markAsDone" to convey that this does the opposite. Furthermore, "undone" is not actually an English word. Perhaps "markAsNotDone" or "unmark" would be better names here.